forge-gui/res/cardsfolder/upcoming/arcee_sharpshooter_arcee_acrobatic_coupe.txt

```diff
@@ -0,0 +1,27 @@
+Name:Arcee, Sharpshooter
+ManaCost:1 R W
+Types:Legendary Artifact Creature Robot
+PT:2/2
+K:More Than Meets the Eye:R W
+K:First Strike
+A:AB$ DealDamage | Cost$ 1 XCantBe0 SubCounter<X/P1P1/NICKNAME> | ValidTgts$ Creature | SubAbility$ DBConvert | SpellDescription$ It deals that much damage to target creature.
+SVar:DBConvert:DB$ SetState | Mode$ Convert | StackDescription$ SpellDescription | SpellDescription$ Convert NICKNAME.
+SVar:X:Count$xPaid
+AlternateMode:DoubleFaced
+DeckHints:Ability$Counters
+Oracle:More Than Meets the Eye {R}{W} (You may cast this card converted for {R}{W}.)\nFirst strike\n{1}, Remove one or more +1/+1 counters from Arcee: It deals that much damage to target creature. Convert Arcee.
+
+ALTERNATE
+
+Name:Arcee, Acrobatic Coupe
+ManaCost:no cost
+Colors:red,white
+Types:Legendary Artifact Vehicle
+PT:2/2
+K:Living metal
+T:Mode$ SpellCast | ValidActivatingPlayer$ You | TargetsValid$ Creature.YouCtrl,Vehicle.YouCtrl | Execute$ TrigPutCounters | TriggerDescription$ Whenever you cast a spell that targets one or more creatures or Vehicles you control, put that many +1/+1 counters on NICKNAME. Convert NICKNAME.
+SVar:TrigPutCounters:DB$ PutCounter | CounterType$ P1P1 | CounterNum$ X | SubAbility$ DBConvert
```

Comment:
Uhh, from the wording I'd say you need to filter the amount too, e.g. a _Decimate_ shouldn't always count as 4

I don't think you'll need new AbilityKey anyway, something with `TriggerObjectsSpellAbilityTargetingCards$Valid` should work...

Reply:
I found more problems:

- `SpellAbilityTargetingCards` only provides the targets of the root, that means you might miss some - currently only _Dack Fayden_ emblem is affected because on some others you used `RememberValidCards$` instead
but it would probably be better to remove that and use the fixed triggeringObject instead (`getAllTargetChoices`)
- The only other place where it gets used is this workaround: de8ff5a0e147cd8d2e27e68379a6614e0490ae78
I tested the case from that but it didn't seem needed, so we should probably remove it so it doesn't mess anything up
@Hanmac can you think of a scenario where we might lose the targets like that?
- `SpellAbilityTarget` for _Zevlor_ could also be broken if the targeting happens on a Sub and should be adressed similarly

Reply:
first where is SpellAbilityTargetingCards really used? github can't show me. and should that be fixed to all targets instead?

The workaround looks ugly, is that really needed?

Reply:
Only _Dack Fayden_ + `AbilityUtils` use it

but cards like _Orvar, the All-Form_ could also be refactored to use it instead of `RememberValidCards` imo

Reply:
yeah from what i can see, they should use all targets ...
BUT probably should put it into a CardCollection to remove duplicates